build.sbt

```diff
@@ -72,3 +72,5 @@ lazy val testSettings: Seq[Def.Setting[_]] = Seq(
 )
 
 dependencyOverrides ++= AppDependencies.overrides
+
+addCommandAlias("scalastyleAll", "all scalastyle test:scalastyle it:scalastyle")
```

Comment:
Contains deprecated code, `in` can be replaced with `/`

Reply:
done